make/brain

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
+NAMESPACE="cf"
+
+has_namespace() {
+    kubectl get namespace --output=name "${NAMESPACE}" >/dev/null 2>/dev/null
+}
+
+stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-brain start
+
+if has_namespace ; then
+    :
+else
+    echo 1>&2 Namespace ${NAMESPACE} is missing. SCF is not running.
+    exit 1
+fi
+
+stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-brain::setup start
+
+. "${GIT_ROOT}/bin/settings/settings.env"
+
+cf api api.cf-dev.io --skip-ssl-validation
+cf auth admin "${CLUSTER_ADMIN_PASSWORD}"
+cf enable-feature-flag diego_docker
+
+stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-brain::setup end
+
+stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-brain::create start
+
+# Delete left-over pod/definition from previous runs, then create/run
+kubectl delete --namespace="${NAMESPACE}" --filename="kube/bosh-task/acceptance-tests-brain.yml" \
+    2> /dev/null || /bin/true
```

Comment:
Yes, we should. Let me know if you want me to fix this, as you already approved the PR!